stdlib/2and3/shutil.pyi

```diff
@@ -23,7 +23,7 @@ elif sys.version_info >= (3,):
     _AnyPath = str
     _PathReturn = str
 else:
-    _Path = unicode
+    _Path = Union[str, unicode]
```

Comment:
nit: Arguments that can be either `bytes` or `unicode` should be annotated as `Text`, according to [CONTRIBUTING.md](https://github.com/python/typeshed/blob/master/CONTRIBUTING.md).

Reply:
Yeah, I agree using Text would be nice. However, I preferred Union because the stdlib documentation for Text is not in sync with what CONTRIBUTING.md file says: https://docs.python.org/3.6/library/typing.html#typing.Text. It seems to me that treating Text as Union[str, unicode] is a Pytype behavior?

Reply:
What I mean is, if I were a user and find Text here, and then refer to the stdlib documentation to learn about what Text is, I am mislead thinking that these functions only accept unicode.

Reply:
I agree that the documentation isn't always consistent, but I assume the typeshed recommendation comes from https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code. (Scroll down to the bottom of that section, where it says, "For parameters typed as `Text`, arguments of type `str` as well as `unicode` should be acceptable.")